pnpm-lock.yaml

```diff
@@ -52,7 +52,7 @@ importers:
       '@changesets/cli': 2.26.2
       '@fluid-private/changelog-generator-wrapper': file:tools/changelog-generator-wrapper
       '@fluid-tools/build-cli': 0.27.0_loebgezstcsvd2poh2d55fifke
-      '@fluid-tools/markdown-magic': link:tools/markdown-magic
+      '@fluid-tools/markdown-magic': file:tools/markdown-magic_loebgezstcsvd2poh2d55fifke
```

Comment:
@Josmithr @tylerbutler I saw the recent PRs related to file/link; looks like the revert somehow missed this?

Reply:
Interesting. I guess `pnpm i` didn't automatically fix that after changing back from `link:` to `file:`?

Reply:
Excuse me, I reverted the PR using github. I haven't had enough coffee yet...

But that presumably means that `pnpm i` didn't migrate from `file:` to `link:` when we made the initial change, which is equally confusing.